storage-account.tf

```diff
@@ -1,135 +1,210 @@
+  aks_env                     = var.env == "sandbox" ? "sbox" : var.env
+
+  aat_cft_vnet_name           = "cft-aat-vnet"
+  aat_cft_vnet_resource_group = "cft-aat-network-rg"
+
+  app_aks_network_name    = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" ? "cft-${local.aks_env}-vnet" : "core-${local.aks_env}-vnet"
+  app_aks_network_rg_name = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" ? "cft-${local.aks_env}-network-rg" : "aks-infra-${local.aks_env}-rg"
+
+  standard_subnets = [
+    data.azurerm_subnet.jenkins_subnet.id,
+    data.azurerm_subnet.jenkins_aks_00.id,
+    data.azurerm_subnet.jenkins_aks_01.id,
+    data.azurerm_subnet.app_aks_00_subnet.id,
+    data.azurerm_subnet.app_aks_01_subnet.id
+  ]
+
+  preview_subnets = var.env == "aat" ? [data.azurerm_subnet.preview_aks_00_subnet.id, data.azurerm_subnet.preview_aks_01_subnet.id] : []
+  valid_subnets   = concat(local.standard_subnets, local.preview_subnets)
+}
+
+data "azurerm_subnet" "preview_aks_00_subnet" {
+  provider             = azurerm.aks-preview
+  name                 = "aks-00"
+  virtual_network_name = local.preview_vnet_name
+  resource_group_name  = local.preview_vnet_resource_group
+}
+
+data "azurerm_subnet" "preview_aks_01_subnet" {
+  provider             = azurerm.aks-preview
+  name                 = "aks-01"
+  virtual_network_name = local.preview_vnet_name
+  resource_group_name  = local.preview_vnet_resource_group
+}
+
+data "azurerm_subnet" "jenkins_subnet" {
+  provider             = azurerm.mgmt
+  name                 = "iaas"
+  virtual_network_name = local.mgmt_network_name
+  resource_group_name  = local.mgmt_network_rg_name
+}
+
+data "azurerm_subnet" "jenkins_aks_00" {
+  provider             = azurerm.mgmt
+  name                 = "aks-00"
+  virtual_network_name = local.mgmt_network_name
+  resource_group_name  = local.mgmt_network_rg_name
+}
+
+data "azurerm_subnet" "jenkins_aks_01" {
+  provider             = azurerm.mgmt
+  name                 = "aks-01"
+  virtual_network_name = local.mgmt_network_name
+  resource_group_name  = local.mgmt_network_rg_name
+}
+
+data "azurerm_subnet" "app_aks_00_subnet" {
+  provider             = azurerm.aks-infra
+  name                 = "aks-00"
+  virtual_network_name = local.app_aks_network_name
+  resource_group_name  = local.app_aks_network_rg_name
+}
+
+data "azurerm_subnet" "app_aks_01_subnet" {
+  provider             = azurerm.aks-infra
+  name                 = "aks-01"
+  virtual_network_name = local.app_aks_network_name
+  resource_group_name  = local.app_aks_network_rg_name
+}
+
 
 // Shared Storage Account
 module "storage_account" {
-  source                   = "git@github.com:hmcts/cnp-module-storage-account?ref=master"
-  env                      = "${var.env}"
+  source                   = "git@github.com:hmcts/cnp-module-storage-account.git?ref=tf_native_versioning"
```

Comment:
this branch should be merged to master, if it's been tested already

Reply:
@timja tf_native_versioning merged to master